src/My_Experiences.html

```diff
@@ -11,22 +11,14 @@
           integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 </head>
 <body style="background-color:whitesmoke; font-family:Helvetica Neue,Helvetica,Arial,sans-serif;">
-
-
-
-
-
```

Comment:
The empty lines in the code could be removed to have overall cleaner code. This can be applied to multiple files. I feel this would be more in line with Baeldung's source file structure section under clean code standards. https://www.baeldung.com/java-clean-code#3-source-file-structure